src/config/mod.rs

```diff
@@ -96,6 +96,22 @@ pub fn safe_app_info_request_timeout() -> u64 {
     u64_with_default("SAFE_APP_INFO_REQUEST_TIMEOUT", 3000)
 }
 
+pub fn transaction_request_timeout() -> u64 {
+    u64_with_default("TRANSACTION_REQUEST_TIMEOUT", 30000)
+}
+
+pub fn safe_info_request_timeout() -> u64 {
+    u64_with_default("SAFE_INFO_REQUEST_TIMEOUT", 10000)
+}
+
+pub fn token_info_request_timeout() -> u64 {
+    u64_with_default("TOKEN_INFO_REQUEST_TIMEOUT", 5000)
```

Comment:
Actually might makes sense to up this to something like 10, 15 or 30

Reply:
I will leave it as 30 to match the clients